SimulationAgent/SimulationRunner.cs

```diff
@@ -127,6 +126,24 @@ public void Start(Services.Models.Simulation simulation)
                 // the active hub, e.g. in case the user provided a custom connection string.
                 this.devices.SetCurrentIotHub();
 
+                // Create the devices
+                try
+                {
+                    var devices = this.simulations.GetDeviceIds(simulation);
+
+                    // This will ignore existing devices, creating only the missing ones
+                    this.devices.CreateListAsync(devices).Wait(TimeSpan.FromSeconds(30));
```

Comment:
why 30s?  Is there a chance you'll end up in the "real world" and need more time?  

Also, nit: move to constant?

Reply:
Moved to constant.
The creation of 1000 devices takes about 2-3 seconds. If it takes longer, this will trigger a timeout exception and the agent will retry. When we go big, we'll refactor/improve this to scale out horizontally.